src/App.jsx

```diff
@@ -2,17 +2,20 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // write code here
+    setCount(currentCount => (
+      currentCount + 1
+    ));
```

Comment:
You can rewrite it like this
```suggestion
    setCount(currentCount => currentCount + 1);
```